webpack/ForemanInventoryUpload/ForemanInventoryHelpers.js

```diff
@@ -2,3 +2,6 @@ import { foremanUrl } from '../ForemanRhCloudHelpers';
 
 export const inventoryUrl = path =>
   foremanUrl(`/foreman_inventory_upload/${path}`);
+
+export const getInventoryDocsUrl = () =>
+  'https://access.redhat.com/products/subscription-central';
```

Comment:
Shouldn't we use foreman's documentation function:

``` js
foremanUrl + '/links/manual?root_url=https://access.redhat.com/products/&section=subscription-central'
```

Reply:
not sure why, it is leading to https://theforeman.org/manuals/2.0/quickstart_guide.html

Reply:
Using this worked:
```js
foremanUrl(
    `/links/manual/+?root_url=${URI.encode(
      'https://access.redhat.com/products/subscription-central'
    )}`
  );
```